frontside/lib/main.dart

```diff
@@ -1,18 +1,23 @@
+import 'package:beamer/beamer.dart';
 import 'package:flutter/material.dart';
-import 'package:frontside/theme.dart';
-import 'package:frontside/views/home/home.dart';
-
+import 'package:frontside/routers/routers.dart';
 void main() {
+  Beamer.setPathUrlStrategy();
   runApp(Sk8());
 }
 
 class Sk8 extends StatelessWidget {
+
+
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'sk8',
-      theme: defaultTheme,
```

Comment:
Where did the `defaultTheme` and `title` go? :D Can you put them back, please? I think they still can be applied under `MaterialApp.router(...)`.